src/posts/secrets-handling-in-kubernetes-using-hashicorp-vault.md

```diff
@@ -0,0 +1,267 @@
+---
+type: Blog
+title: Secrets handling in Kubernetes using HashiCorp Vault
+subheading: In this blog we explore several approaches for getting secrets from HashiCorp Vault into Jenkins running in Kubernetes
+authors:
+- mvainio
+tags:
+- Kubernetes
+- Vault
+- Jenkins
+date: 2022-10-12
+image: "/blogs/secrets-handling-in-kubernetes-using-hashicorp-vault/secrets-handling-in-kubernetes-using-hashicorp-vault.png"
+featured: true
+
+---
+
+When running Jenkins jobs using the Kubernetes plugin, there are many ways to fetch secrets from HashiCorp Vault. [In the previous post](https://verifa.io/blog/secrets-handling-in-kubernetes-a-jenkins-story/) we stored the secrets in Kubernetes, but let’s look at options that don’t persist secrets in the cluster. 
```

Comment:
I think we need to drop the `https://verifa.io` from the links that stay on the site. Right @jlarfors?

```
# before:
https://verifa.io/blog/secrets-handling-in-kubernetes-a-jenkins-story/
# after:
/blog/secrets-handling-in-kubernetes-a-jenkins-story/
```

Reply:
ah, just had a quick look into this and yes internal links shouldn't open a new tab (better for UX, SEO, etc etc). Only external links to open in a new tab.